PBAB-QUICK-START.md

```diff
@@ -0,0 +1,26 @@
+# Gain Access to Twitter and Discord APIs
```

Comment:
Let's link out to this new `.md` file in https://github.com/ArtBlocks/alertbot#supporting-your-pbab-project